pystan/experimental/__init__.py

```diff
@@ -1,6 +1,7 @@
-import logging
-from .misc import *
+import logging as _logging
+from .misc import fix_include
+from .pickling_tools import load_fit
 
-logger = logging.getLogger('pystan')
+_logger = _logging.getLogger('pystan')
```

Comment:
Would ``from pystan import logger`` work?

Reply:
`getLogger` is probably right thing to do. I'm just confused about why there are the added underscores. They don't seem necessary.

Reply:
Yes, I didn't want pollute the namespace because the submodule has only 2 main functions. I can change it back.

Reply:
We don't hide `logging` anywhere else. That's what looked weird to me.

If you want to be very economical:
```
import logging
logging.getLogger('pystan').warning('...')
```